payments/swish/seamless-view.md

```diff
@@ -21,112 +21,41 @@ sidebar:
-                          within your webshop." %}
+                          within your webshop. This gives the consumer a
+                          frictionless experience as we are handling the
+                          payment in the implemented `iframe` on your page" %}
 
-## Introduction
+## Swish Seamless View integration flow
 
-The Seamless View integration provide you with the Swish payment solution
-directly on your website. This gives the consumer a frictionless experience as
-we are handling the payment in the implemented `iframe` on your page (see
-example below).
-
-![screenshot of the seamless view swish payment
-page][seamless-view-img]{:height="250px" width="660px"}
-
-## Purchase flow
-
-```mermaid
-sequenceDiagram
-    participant Payer
-    participant Merchant
-    participant SwedbankPay as Swedbank Pay
-
-    activate Payer
-    Payer->>-Merchant: start purchase
-    activate Merchant
-    note left of Payer: First API request
-    Merchant->>-SwedbankPay: POST /psp/swish/payments ①
-    activate SwedbankPay
-    SwedbankPay-->>-Merchant: rel: view-sales ②
-    activate Merchant
-    Merchant-->>-Payer: confirmation page
-    activate Payer
-    note left of Payer: Open iframe ③
-    Payer->>Payer: Input mobile number
-    Payer->>-SwedbankPay: Show Consumer UI page in iframe - Authorization ④
-    activate SwedbankPay
-        opt Card supports 3-D Secure
-        SwedbankPay-->>-Payer: redirect to IssuingBank
-        activate Payer
-        Payer->>IssuingBank: 3-D Secure authentication process
-        activate IssuingBank
-        IssuingBank->>-Payer: 3-D Secure authentication process
-        Payer->>-SwedbankPay: access authentication page
-        end
-    SwedbankPay-->>Merchant: Event: OnPaymentComplete ⑤
-    activate Merchant
-    note left of Merchant: Second API request.
-    Merchant->>-SwedbankPay: GET <payment.id> ⑥
-    activate SwedbankPay
-    SwedbankPay-->>-Merchant: rel: view-sales
-    activate Merchant
-    Merchant-->>-Payer: display purchase result
-    activate Payer
-
-        opt Callback is set ⑦
-        activate SwedbankPay
-        SwedbankPay->>SwedbankPay: Payment is updated
-        SwedbankPay->>-Merchant: POST Payment Callback
-        end
-```
-
-### Explainations
-
-* ① When the payer starts the purchase process, you make a `POST` request
+1. When the payer starts the purchase process, you make a `POST` request
   towards Swedbank Pay with the collected Purchase information.
-* ② `view-sales` is a `rel` value in one of the operations, sent as a response
+  `view-sales` is a `rel` value in one of the operations, sent as a response
   from Swedbank Pay to the Merchant.
-* ③ `Open iframe` creates the Swedbank Pay hosted iframe.
-* ④ `Show Consumer UI page in iframe` displays the payment window as content
+1. `Open iframe` creates the Swedbank Pay hosted iframe.
+1. `Show Consumer UI page in iframe` displays the payment window as content
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[ordered-list-marker-value] Marker should be `3`, was `1`](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5210590)